### PR TITLE
chore: fix downloads

### PR DESCRIPTION
**Pull request by Cifko**
- Change the order of the releases from newest to oldest ( (by `lastModified`).
- The download button now use the latest (by `lastModified`) from the `latest` releases.

### CODE REVIEW

File: assets/js/downloads.js [hunk 1]
```diff
@@ -93,14 +94,15 @@ jQuery(document).ready(function ($) {
         let checkSumDiv = document.getElementById(`${rawOs}CSID`);
 
         let sha256 = "";
-        let checksum = data[os][0].sha256;
+        let latest = data[os].reduce((a, b) => a.lastModified > b.lastModified || (a.lastModified == b.lastModified && a.url < b.ulr) ? a : b)
```
Comment (delta1) [inline assets/js/downloads.js]: `ulr` ? 

if this typo doesn't error then is that branch of the conditional necessary?

Comment (Cifko) [reply assets/js/downloads.js]: It's there to distiguish between the zip file and the checksum file. I'll add a comment, or do you have a more elegant way to check?

Comment (delta1) [reply assets/js/downloads.js]: okay please add a comment and fix the typo 👍

Comment (Cifko) [reply assets/js/downloads.js]: done

**Review by delta1 — APPROVED** [review]
Awesome! Thanks @Cifko

**Review by stringhandler — APPROVED** [review]

**Pull request merged**